docs/modules/reference/pages/service-assurance/monitors/MailTransportMonitor.adoc

```diff
@@ -315,19 +352,37 @@ Test that we can connect via IMAPS and open the OPENNMS folder.
   <parameter key="mail-transport-test">
     <mail-transport-test>
       <mail-test>
-        <readmail-test attempt-interval="5000" mail-folder="OPENNMS" debug="false" >
-          <readmail-host host="imap.gmail.com" port="993">
-            <readmail-protocol ssl-enable="true" start-tls="false" transport="imaps" />
+        <readmail-test attempt-interval="5000" <1>
+                       mail-folder="OPENNMS" <2>
+                       debug="false" > <3>
+          <readmail-host host="imap.gmail.com" <4>
+                         port="993"> <5>
+            <readmail-protocol ssl-enable="true" <6>
+                               start-tls="false" <7>
+                               transport="imaps" /> <8>
           </readmail-host>
-          <user-auth user-name="opennms@gmail.com" password="opennms"/>
+          <user-auth user-name=opennms@gmail.com <9>
+                     password="opennms"/> <10>
         </readmail-test>
       </mail-test>
     </mail-transport-test>
   </parameter>
-  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/>
-  <parameter key="ds-name" value="rdmail_lat"/>
+  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/> <11>
+  <parameter key="ds-name" value="rdmail_lat"/> <12>
 </service>
 
-<monitor service="MTM-Readmail" class-name="org.opennms.netmgt.poller.monitors.MailTransportMonitor" /> <1>
+<monitor service="MTM-Readmail" class-name="org.opennms.netmgt.poller.monitors.MailTransportMonitor" /> <13>
 ----
-<1> Required *monitor* section.
+<1> Interval in milliseconds between send attempts.
+<2> The folder or IMAP label to check for mail.
+<3> Show additional debug output.
+<4> The SMTP server address for sending mail.
+<5> The SMTP server port.
+<6> Whether to enable SSL for the connection.
+<7> Use the STARTTLS command
+<8> The transport protocol to use: pop3, pop3s, imap, or imaps.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.